src/main/java/org/jabref/logic/journals/JournalAbbreviationRepository.java

```diff
@@ -67,7 +69,30 @@ public boolean isAbbreviatedName(String journalName) {
         String journal = journalName.trim();
 
         return customAbbreviations.stream().anyMatch(abbreviation -> isMatchedAbbreviated(journal, abbreviation))
-                || abbreviationToFull.containsKey(journal);
+                || abbreviationToFull.containsKey(journal) || findDottedAbbrFromDotless(journal).length() > 0;
+    }
+
+    public String findDottedAbbrFromDotless(String journalName) {
+        String foundKey = "";
+
+        // check for a dot-less abbreviation
+        if (!Pattern.compile("\\.").matcher(journalName).find()) {
```

Comment:
If you use Pattern.compile it's usually a good idea to store the pattern as a static final variable e.g. 
`final static Pattern DOT = Pattern.compile("\\."); `

Reply:
Static variable has been set up.